huaweicloud/services/ddm/data_source_huaweicloud_ddm_flavors.go

```diff
@@ -287,5 +287,5 @@ func buildGetDdmFlavorsQueryParams(d *schema.ResourceData, offset int) string {
 
 func updatePathOffset(path string, offset int) string {
 	index := strings.Index(path, "offset")
-	return fmt.Sprintf("%s&offset=%v", path[:index], offset)
+	return fmt.Sprintf("%soffset=%v", path[:index], offset)
```

Comment:
it depends on the `offset` order in the function buildGetDdmFlavorsQueryParams. 
pls Decouple it

Reply:
in the package, buildGetDdmFlavorsQueryParams can make sure that `offset` is at the last  of query parameters, so I think it looks good.

Reply:
the param has already been placed at the end in the func buildGetDdmFlavorsQueryParams